fission-cli/library/Fission/CLI/Remote.hs

```diff
@@ -19,6 +20,14 @@ import           Fission.CLI.Remote.Class
 getRemoteBaseUrl :: MonadRemote m => m BaseUrl
 getRemoteBaseUrl = toBaseUrl <$> getRemote
 
+getIpfsGateway :: MonadRemote m => m String
+getIpfsGateway = do
+  url <- getRemoteBaseUrl
+  let BaseUrl {..} = url
+      ipfsGateway = url { baseUrlHost = "ipfs." <> baseUrlHost, baseUrlPath = "ipns" }
+
+  pure $ showBaseUrl ipfsGateway
```

Comment:
Sorry, this is just nitpicking now 😛 

```haskell
getIpfsGateway = do
  url@BaseUrl {..} <- getRemoteBaseUrl
  return $ showBaseUrl url { baseUrlHost = "ipfs." <> baseUrlHost, baseUrlPath = "ipns" }
```

Reply:
Yup remembered how to do this when I was falling asleep last night. At first I tried the elixir version `BaseUrl{..} = url <-` which didn't compile for obvious reasons so I just did the dumb thing :shrug: Thanks for pointing it out anyways!